src/js/views/patients/patient/dashboard/dashboard_views.js

```diff
@@ -95,6 +96,9 @@ const ListView = CollectionView.extend({
   tagName: 'table',
   childView: ItemView,
   emptyView: EmptyView,
+  viewComparator({ model }) {
+    return - moment(model.get('updated_at')).format('X');
+  },
```

Comment:
did this because the other way broke the 'new' action (since `model.moment` isn't a thing on the 'blank' new action?)